Makefile

```diff
@@ -24,6 +24,22 @@ env:
 	@rm -rf .env
 	@./env.sh
 
+# help:
+.PHONY:	tf-deploy
```

Comment:
@gmflau How do you feel about making this `make` command something like: `make redis-deploy`? so it's clear what's being deployed by TF.

Reply:
Yeah.  redis-deploy and redis-destroy would work.